src/schema/posts.ts

```diff
@@ -488,7 +498,7 @@ export const typeDefs = /* GraphQL */ `
   }
 
   type UpvoteEdge {
-    node: Upvote!
+    node: UserPost!
```

Comment:
This is a breaking change, to make it backwards compatible we need to introduce `createdAt` field which will be identical to `votedAt`

Reply:
@idoshamun from what I see we don't use said `createdAt` anywhere in webapp or extension. 

Also the said `UpvoteEdge` is only used for `postUpvotes` query for sorting, test is confirmed and updated [here](https://github.com/dailydotdev/daily-api/pull/1398/files#diff-6e46397f1e2cc9f81f6d251217c5f625accc5596fbd5e11dcdb644922166ddb9R919-R958). 

So even though its breaking change it should not affect any current client (aside from updating types to prevent using wrong field in the future [as mentioned here](https://github.com/dailydotdev/daily-api/pull/1398#discussion_r1297372829)).

To support this breaking change we would need to introduce the:
- GraphORMType
- this type would use `from` to pull from `user_post` table
- we would have a `createdAt` fields override with `select: 'votedAt'` (since `createdAt` also exists for `user_post`)
  - this will also be confusing I think
- then we would use that type just for this one case
- it would require maintenance down the line even though we completely move to new vote design

So if there is not easier way to do the above I think we can go with this breaking change.

cc: @rebelchris